Staccato-iOS/Staccato-iOS/Presentation/Components/StaccatoNavigationBar.swift

```diff
@@ -35,17 +42,19 @@ struct StaccatoNavigationBar<T: View>: ViewModifier {
                             .foregroundStyle(.gray2)
                     }
 
-                    VStack(alignment: .leading) {
-                        if let title {
-                            Text(title)
-                                .typography(.title2)
-                                .foregroundStyle(.gray5)
-                        }
-
-                        if let subtitle {
-                            Text(subtitle)
-                                .typography(.body4)
-                                .foregroundStyle(.gray5)
+                    if titlePosition == .leading {
```

Comment:
titlePosition이 enum이니까 이 부분은 switch 문으로 만들면 어떨까요?

Reply:
흠 조금 고민이 되는 지점이네요. 이 경우에는 한 케이스가 아닌 경우에는 처리하지 않아도 되는 로직인데, switch 문을 사용했을 경우에는 EmptyView()로 처리를 해야할 것 같아요.

`명시적으로 Case를 관리` vs `가독성` 어떤 것을 챙기는 것이 나을까요? 의견을 나누면 좋을 것 같아요!

Reply:
수정 제안을 해주셔도 좋아요! ㅎㅎ

Reply:
switch문을 사용할 경우 EmptyView 처리를 해야한다는 게 무슨 뜻인지 잘 모르겠어요🥺

Reply:
```suggestion
                    switch titlePosition {
                    case .center:
                        EmptyView()
                    case .leading:
                        // leading에 해당하는 로직
                    }
```

이렇게 바뀌어야 한다는 말이었어요. 해당되지 않는 케이스에는 추가적으로 EmptyView()로 표시를 해주어야 하기 때문에, 오히려 가독성이 떨어지지 않을까? 란 생각이었습니다.